use std::error::Error;

pub async fn get_completion(prompt: &str) -> Result<String, Box<dyn Error>> {
    // TODO: 实现 OpenAI API 调用逻辑
    Ok(String::new())
} 